README.md

```diff
@@ -157,6 +157,32 @@ xtf.foo.v2.version=1.0.3
 
 Retrieving an instance with this metadata: `Produts.resolve("product");`
 
+#### Run test cases in separate namespaces using `xtf.openshift.namespace.per.testcase` property 
+
+This feature requires to have XTF JUnit5 `cz.xtf.junit5.listeners.ProjectCreator` extension enabled. This can be done by adding
```

Comment:
Do we need to duplicate here how to enable ProjectCreator, shouldn't we just link to ProjectCreator section?

Reply:
Will you create it?

Reply:
yes :-)